### PR TITLE
Update theme-create.md

### PR DESCRIPTION
**Pull request by Serj190336**
## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@Serj190336 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by Serj190336** [comment]
word fix

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Hi @Serj190336. Thank you for your collaboration. Please, sign [Adobe CLA](http://opensource.adobe.com/cla.html). It's a required one-time operation for processing the changes introduced in your PRs. 

Thank you!

File: guides/v2.3/frontend-dev-guide/themes/theme-create.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The folder name conventionally matches naming used in the theme's code: any alph
 
 ## Declare your theme {#fedg_create_theme_how-to_declare}
 
-After you create a directory for your theme, you must create `theme.xml` containing at least the theme name. Optionally, you can specifiy the parent theme name (if the theme [inherits]({{page.baseurl}}/frontend-dev-guide/themes/theme-inherit.html) from one) and, if necessary, where the theme preview image is stored.
+After you create a directory for your theme, you must create `theme.xml` containing at least the theme name. Optionally, you can specify the parent theme name (if the theme [inherits]({{page.baseurl}}/frontend-dev-guide/themes/theme-inherit.html) from one) and, if necessary, where the theme preview image is stored.
```
Comment (rogyar) [inline guides/v2.3/frontend-dev-guide/themes/theme-create.md]: Could you also fix the same spelling issue in `guides/v2.2/frontend-dev-guide/themes/theme-create.md`, please?

**Comment by rogyar** [comment]
Also, I would kindly ask you do the following for your future pull requests:

## Provide a meaningful PR title.

### Bad example:

```
Edit <file>
Update <file>
Minor fixes
```

### Good example:

```
Fixed spelling issue in the "Create New Theme" topic
Provided additional information about scopes Magento Cloud environment variables
Fixed incorrect order of layout nodes injection of checkout UI components
```

## Fill in the template for PR description

The  template is introduced by default, once a PR creation process is initialized on Github

### Bad practice

- Remove everything, write something like "fix for <file>"
- Leave all sections empty

### Good practice

- Using the provided template, provide a short description of the purpose of your PR
- Provide a list of affected pages in the corresponding section
- Provide links to the source code if it's applicable for your PR

Thank you!

**Comment by dobooth** [comment]
Hi @Serj190336 Thanks for the contribution. We recently updated our CLA and need to have all contributors sign the new version. If you could do that below, we can process this PR. Thanks!

**Pull request merged**